src/pages/Models/Models.js

```diff
@@ -35,14 +36,14 @@ export default function Models() {
               "alert"
             )}, ${running} running`}
           </div>
-          <ButtonToggle />
+          <ModelGroupToggle setGroupedBy={setGroupedBy} groupedBy={groupedBy} />
```

Comment:
At some point in the future we'll probably want to persist this in a query string.

Reply:
https://github.com/canonical-web-and-design/jaas-dashboard/issues/261